templates/base/src/utils/Logger/logger.js

```diff
@@ -0,0 +1,125 @@
+
+import fetch from 'isomorphic-unfetch';
+
+/**
+ * USAGE: ->
+ * import Logger from '../utils/Logger';
+ *  call as :-
+ * Logger.error({ message: 'Here is the error',
+ *    event: {
+      name: 'event20', id: 657,
+      module: 'modA',
+      type: "load|interaction|redirect" ,
+      data: { a: "hhhh" },
+    },
+    error: {
+      code: 'ABC',
+      message: 'message',
+      operationName: 'testing'
+    },
+    service: {
+      name: 'serviceName',
+      path: 'servicePath',
+    } });
+ */
+
+/**
+ * @param {object} msg --> msg Object
+ * @param {string}  logLevel --> string (info, log, error)
+ * returns an object with log data to be sent in api
+ */
+const getLogObject = ({ msg, logLevel }) => {
+  if (process.browser) {
```

Comment:
@all9planets this way we will end up creating 2 completely diff objects for server and client. We need to conditionally exclude or set empty objects.

Reply:
done